CHANGELOG.md

```diff
@@ -1 +1,67 @@
+
+### Deprecated
+* Remove deprecated integration-cli/environment functions [moby/moby#36018](https://github.com/moby/moby/pull/36018)
+
+### Experimental
+
++ Add manifest command [docker/cli#138](https://github.com/docker/cli/pull/138)
+
+### Logging
+
+* Added tag log option to json-logger and use RawAttrs [moby/moby#35949](https://github.com/moby/moby/pull/35949)
+* Improve daemon config reload; log active configuration [moby/moby#36019](https://github.com/moby/moby/pull/36019)
+* Wrap response error for container logs method. [moby/moby#36000](https://github.com/moby/moby/pull/36000)
++ Add journald tag as SYSLOG_IDENTIFIER [moby/moby#35570](https://github.com/moby/moby/pull/35570)
+* Splunk: limit the reader size on error responses [moby/moby#35509](https://github.com/moby/moby/pull/35509)
+
+### Networking
+
+* Alias container and network packages to stop name clashes [moby/moby#36061](https://github.com/moby/moby/pull/36061)
+* Disable service on release network [moby/moby#35960](https://github.com/moby/moby/pull/35960)
+* Rename FindUniqueNetwork to FindNetwork [moby/moby#36021](https://github.com/moby/moby/pull/36021)
+* Update `FindNetwork` to address network name duplications [moby/moby#30897](https://github.com/moby/moby/pull/30897)
+- Fix --network-add adding duplicate networks [docker/cli#780](https://github.com/docker/cli/pull/780)
+- Fixing ingress network when upgrading from 17.09 to 17.12. [moby/moby#36003](https://github.com/moby/moby/pull/36003)
+- Fix ndots configuration [docker/libnetwork#1995](https://github.com/docker/libnetwork/pull/1995)
+* Enable IPV6 config on Sandbox creation on live-restore [docker/libnetwork#2043](https://github.com/docker/libnetwork/pull/2043)
+* Handle DNS queries of type MX [docker/libnetwork#2041](https://github.com/docker/libnetwork/pull/2041)
+
+### Runtime
+
+* Bump to RS3 final build, and remove LCOW_SUPPORTED [moby/moby#36065](https://github.com/moby/moby/pull/36065)
+* LCOW remotefs - return error in Read() implementation [moby/moby#36051](https://github.com/moby/moby/pull/36051)
+* LCOW: Coalesce daemon stores, allow dual LCOW and WCOW mode [moby/moby#34859](https://github.com/moby/moby/pull/34859)
+* LCOW: Fix OpenFile parameters [moby/moby#36043](https://github.com/moby/moby/pull/36043)
+* Revendor Microsoft/go-winio @ v0.4.6 [moby/moby#35968](https://github.com/moby/moby/pull/35968)
+* Vendor Microsoft/hcsshim @ v0.6.8 [moby/moby#35924](https://github.com/moby/moby/pull/35924)
+* Windows: Bump RW layer size [moby/moby#35925](https://github.com/moby/moby/pull/35925)
+* Devicemapper: ignore Nodata errors when delete thin device [moby/moby#35919](https://github.com/moby/moby/pull/35919)
+* Introduce « exec_die » event [moby/moby#35744](https://github.com/moby/moby/pull/35744)
+* Ensure CPU quota/period updates are sent to runc [moby/moby#36030](https://github.com/moby/moby/pull/36030)
```

Comment:
Fix `docker update` not updating cpu quota, and cpu-period of a running container